private-templates-service/client/src/components/AdaptiveCardPanel/AdaptiveCardPanel.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import { Container, TemplateName, ACWrapper } from "./styled";
+import AdaptiveCard from '../Common/AdaptiveCard';
+import { renderAdaptiveCard } from "../Common/AdaptiveCard/AdaptiveCard";
+
+interface Props {
+  toggleModal: () => void;
+}
+
+class AdaptiveCardPanel extends React.Component<Props> {
+  render() {
+    return (
+      <Container>
+        <ACWrapper>
+          <AdaptiveCard />
```

Comment:
remove space between AdaptiveCard and />

Reply:
It is standard to have a space when using self-closing tags 
![image](https://user-images.githubusercontent.com/19780117/73896748-c0e7b300-4838-11ea-8c20-b59d6dd1322d.png)

Reply:
:o